go.mod

```diff
@@ -1,12 +1,19 @@
 module github.com/xgfone/bttools
 
 require (
-	github.com/urfave/cli/v2 v2.2.0
+	github.com/cpuguy83/go-md2man/v2 v2.0.0 // indirect
+	github.com/russross/blackfriday/v2 v2.1.0 // indirect
+	github.com/samuel/go-zookeeper v0.0.0-20201211165307-7117e9ea2414 // indirect
+	github.com/urfave/cli/v2 v2.3.0
 	github.com/xgfone/bt v0.4.0
 	github.com/xgfone/gconf/v5 v5.0.0
-	github.com/xgfone/goapp v0.16.0
+	github.com/xgfone/goapp v0.17.0
 	github.com/xgfone/gover v0.3.0
-	golang.org/x/net v0.0.0-20190613194153-d28f0bde5980
+	golang.org/x/net v0.0.0-20210119194325-5f4716e94777
+	golang.org/x/sys v0.0.0-20210124154548-22da62e12c0c // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
 
 go 1.11
+
+replace github.com/xgfone/bt v0.4.0 => ../bt
```

Comment:
1. The module `bt` has released the version `v0.4.1`, and you can use it. It is no need to use the directive `replace` to replace the module `bt`.
2. You can only update the module `bt` to `v0.4.1`, not all. Or use the command `go get $MODULE`, such as `go get github.com/urfave/cli/v2` to update every module. The advantage is that it won't `require` the indirect modules in `go.mod`.
3. Run the command `go mod tidy` to update the file `go.sum`.